docs/content/any/guides/roles/_index.md

```diff
@@ -18,6 +18,14 @@ These guides will help you add roles to your application and use them to perform
 To learn more about roles and the different ways they can be used for authorization, head over to [Role-Based Access
 Control](learn/roles).
 
+{{% callout "Implementing roles?" "yellow" %}}
+We're working on a set of new features to better support adding roles to B2B applications.
```

Comment:
```suggestion
We're working on the next major release of Oso, a new set of features to better support adding roles to B2B applications.
```

Reply:
The phrase "major release" has a specific meaning in the semver world — I could see people misintepreting this as "we're working on Oso 1.0, a new set..."

Reply:
Is "major update to Oso" better? Or same problem?

Reply:
How about this?

```suggestion
We're working on the next major feature of Oso, a new set of tools to better support adding roles to B2B applications.
```